### PR TITLE
tests: drop extra [Serial] string from spec name

### PR DESCRIPTION
**Pull request by akalenyu**
### What this PR does
This is not needed anymore as ginkgo's serial decorator
adds a label now too: https://github.com/onsi/ginkgo/pull/1484

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -436,11 +436,9 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} && -z ${label_filter}
     label_filter='(sig-compute-migrations && !(GPU,VGPU))'
   elif [[ $TARGET =~ sig-compute-serial ]]; then
     export KUBEVIRT_E2E_PARALLEL=false
-    ginko_params="${ginko_params} --focus=\\[Serial\\]"
-    label_filter='((sig-compute) && !(GPU,VGPU,sig-compute-migrations))'
+    label_filter='((sig-compute,Serial) && !(GPU,VGPU,sig-compute-migrations))'
```
Comment (brianmcarey) [inline automation/test.sh:439]: Should this be?
```suggestion
    label_filter='((sig-compute && Serial) && !(GPU,VGPU,sig-compute-migrations))'
```

**Review by brianmcarey — COMMENTED** [review]
This is great to get the Serial label added - thanks @akalenyu 

Just one question on the label filters for the sig-compute-serial lane.

File: automation/test.sh (hunk 1 above)
Comment (akalenyu) [reply automation/test.sh:439]: Interesting, I'm honestly not sure why this works as is

Comment (brianmcarey) [reply automation/test.sh:439]: I think `,` is an OR operation so it will run all sig-compute and all serial ... it will take a while.

Comment (akalenyu) [reply automation/test.sh:439]: You're right my bad

**Comment by akalenyu** [comment]
There still seems to be some disrepancy between the serial run in this PR and the previous merge:
`Will run 188 of 1431 specs`
While this normally is 184

**Comment by akalenyu** [comment]
> There still seems to be some discrepancy between the serial run in this PR and the previous merge: `Will run 188 of 1431 specs` While this normally is 184

See the last commit for an explanation of the discrepancy

**Review by fossedihelm — COMMENTED** [review]
Looks wonderful!
Can we adjust the `tests/README.md` too?
Thank you

**Comment by akalenyu** [comment]
> Looks wonderful! Can we adjust the `tests/README.md` too? Thank you

done

**Review by EdDev — APPROVED** [review]
This is great! Thanks!

**Comment by iholder101** [comment]
Awesome stuff @akalenyu! Kudos!!

/approve

**Comment by akalenyu** [comment]
@EdDev @fossedihelm small conflict in tests/network/bindingplugin.go due to https://github.com/kubevirt/kubevirt/commit/581ee68c7868cd7ea2e575d7a16ba02f4ceeb701#diff-6908ec12a3bf74f9420e69e25c52eefee7906bfe638fbf10a1718940ecafdff0

**Review by dhiller — APPROVED** [review]
Kudos @akalenyu for enabling this in onsi/ginkgo and also for not shying away from the definitely tedious work of cleaning up the tests!

:tada: :bow: :pray:

**Comment by iholder101** [comment]
/lgtm

Sunday is a good time to avoid conflicts hopefully :) :crossed_fingers:

**Comment by akalenyu** [comment]
> @EdDev @fossedihelm small conflict in tests/network/bindingplugin.go due to [581ee68#diff-6908ec12a3bf74f9420e69e25c52eefee7906bfe638fbf10a1718940ecafdff0](https://github.com/kubevirt/kubevirt/commit/581ee68c7868cd7ea2e575d7a16ba02f4ceeb701#diff-6908ec12a3bf74f9420e69e25c52eefee7906bfe638fbf10a1718940ecafdff0)

yet another rebase. similar conflicts

**Pull request merged**